kitchen.yml

```diff
@@ -10,7 +10,7 @@ transport:
 
 provisioner:
   name: chef_zero
-  deprecations_as_errors: true
+  deprecations_as_errors: false
```

Comment:
I'm pretty sure we can remove the dependency on the windows cookbook if we're bumping unified mode which would fix this. Can you please do the following?
- Revert this to `true`
- Remove the windows dependency in `metadata.rb`
- Bump the required chef version to `>= 15.3` in `metadata.rb`

Reply:
Pretty close, manager looks like the only resource that needs love.